arches/app/views/api/__init__.py

```diff
@@ -233,7 +230,6 @@ def get(self, request):
             tiles = tiles.filter(resourceinstance_id__in=resourceid.split(","))
         if tileid is not None:
             tiles = tiles.filter(tileid=tileid)
-        tiles = tiles.order_by("sortorder")
```

Comment:
We can leave these in if people prefer the explicitness. (Although there is something to be said for know-your-model 😉 )